spec/models/database_spec.rb

```diff
@@ -104,9 +99,6 @@ def generate_migration(model_class, foreign_key_table_name, foreign_key_column)
     migration_name = "add_foreign_key_to_#{model_class.table_name}_" \
                      "#{foreign_key_table_name}_#{foreign_key_column}"
     migration_class_name = migration_name.camelize
-    millisecond_timestamp = generate_timestamp
```

Comment:
You should also delete `generate_timestamp` proc below

Reply:
> I feel like database_spec.rb should have been a rake file instead of a spec file.

Yes, it should have been a gem with a rake file and an easy API for Rspec. But since we wanted to check for needed foreign keys in specs, we did the most simple thing and put it all there.

Reply:
> you now have to manually create the migration

Yes. But we haven't needed that yet. Maybe we will never need it if we always think of `t.references` with `foreign_key: true`. Rails makes it pretty simple to get right in the beginning. And the failing specs can tell you when you introduce a new reference so that you'll never need a migration afterwards.

Reply:
> You should also delete generate_timestamp proc below

Of course, thank you. Done now.